prow/jobs/incubator/compass-console/compass/compass-ui.yaml

```diff
@@ -58,6 +58,56 @@ presubmits: # runs on PRs
           - name: config
             configMap:
               name: kaniko-build-config
+    - name: pull-compass-console-lint
+      annotations:
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pull-compass-console-lint"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      run_if_changed: '^compass/|^components/|^shared/|^package.json'
+      optional: false
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/buildpack-node:v20221028-f16e46a1"
+            command:
+              - "/bin/sh"
+            args:
+              - "-c"
+              - "./scripts/prow-lint.sh"
```

Comment:
Why did you decide to run shell script here? It's not a problem here, but I would like to know the requirements you need to meet with those jobs. Especially because we have golangci-lint image that we centrally managed in test-infra.

Reply:
we need `npm install eslint-plugin-jsdoc@15.12.2`

Reply:
Ah, makes sense.